### DISCUSSION TITLE
New V200 no sensors bug

### DISCUSSION THREAD

**Discussion by weltraumduft** [Rem0o/FanControl.Releases#2705] [Q&A]
Hello, after the new update to V200 this great Software doesnt work. I have no sensors detected, no control.
Now i have install V199 again and all works fine.

I use Windows 11, on AsRock x690 Aqua OC

![Screenshot 2024-09-04 161327](https://github.com/user-attachments/assets/99fd83b5-9fe3-45da-abd4-d6093ed2d3b0)

**Comment by Rem0o** [comment]
Any log.txt?

**Reply by weltraumduft** [reply]
Yes Sir, i give you :)
Hope i can support to find the problem. Tell me what you need.

[log.txt](https://github.com/user-attachments/files/16881822/log.txt)

**Reply by Rem0o** [reply]
HidSharp.Exceptions.DeviceIOException: Failed to get info.
   at HidSharp.Platform.Windows.WinHidDevice.RequiresGetInfo(GetInfoFlags flags)
   at HidSharp.Platform.Windows.WinHidDevice.GetSerialNumber()
   at LibreHardwareMonitor.Hardware.Controller.AquaComputer.MPS..ctor(HidDevice dev, ISettings settings)


>  LibreHardwareMonitor.Hardware.Controller.AquaComputer.

Disable the "controller" section in sensor sources/settings.

**Reply by weltraumduft** [reply]
But this will disable my Water Temp from the Aqua Computer Sensors. Without this i have no control about the MIX curves and Pumps.

**Reply by weltraumduft** [reply]
Hi again :)
I have install the new Version V201 and the same bug. From 110 sensors only 11 founded. Then i disabled the "controller" section. After that all sensors was found, but only 3 important sensors for AquaComputer is missing.

> at HidSharp.Platform.Windows.WinHidDevice.GetSerialNumber()

It is a licence problem?
Thanks and have a nice week.

**Reply by weltraumduft** [reply]
Just want to say, the problem solved after install the new V203
Good work @Rem0o thanks :)
